exporting/opentsdb/opentsdb.c

```diff
@@ -192,6 +246,42 @@ static inline void opentsdb_build_message(BUFFER *buffer, char *message, const c
         message);
 }
 
+/**
+ * Format host labels for OpenTSDB HTTP connector
+ *
+ * @param instance an instance data structure.
+ * @param host a data collecting host.
+ * @return Always returns 0.
+ */
+int format_host_labels_opentsdb_http(struct instance *instance, RRDHOST *host)
+{
+    if (!instance->labels)
+        instance->labels = buffer_create(1024);
+
+    if (unlikely(!sending_labels_configured(instance)))
+        return 0;
+
+    netdata_rwlock_rdlock(&host->labels_rwlock);
+    for (struct label *label = host->labels; label; label = label->next) {
+        if (!should_send_label(instance, label))
+            continue;
+
+        char escaped_value[CONFIG_MAX_VALUE * 2 + 1];
+        sanitize_json_string(escaped_value, label->value, CONFIG_MAX_VALUE);
```

Comment:
The variable `escaped_value` has two times the size given as argument, should we not have a bigger argument length?

Reply:
Already answered.